README.md

```diff
@@ -1317,6 +1317,7 @@ listed below.
 - Under section **(1) Check the IP Whitelist**, click on **ALLOW ACCESS FROM ANYWHERE**. The form will add a field with `0.0.0.0/0`.  Click **SAVE** to save the `0.0.0.0/0` whitelist.
 - Under section **(2) Choose a connection method**, click on **Connect Your Application**
 - In the new screen, click on **Standard connection string (3.4+ driver)**. __*WARNING*__: Do not pick 3.6+ since there Express Session currently has a compatibility issue with it.
```

Comment:
Should this line be removed?

Reply:
@YasharF  Yes, the 3.4+ driver is not present at all in the interface anymore

Reply:
Thanks, can you include removing it in your PR?

Reply:
Updated it

Reply:
The PR isn't showing that the line is removed yet.

Reply:
Sorry, missed it. I have now updated it.